### PR TITLE
Telescope extension Doc

### PR DESCRIPTION
**Pull request by ZieMcd**
Added a bit to documentation about getting telescope extension to work on Ubuntu and Debian systems

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,12 +214,14 @@ require("telescope").extensions.rest.select_env()
 - env_pattern: For env file pattern
 - env_edit_command: For env file edit command
 
+If running Ubuntu or Debian based systems you might need to run `ln -s $(which fdfind) ~/.local/bin/fd` to get extension to work. This becuase extension runs the [fd](https://github.com/sharkdp/fd?tab=readme-ov-file#installation) command.
```
Comment (NTBBloodbath) [inline README.md:217]: There's a small typo in `This becuase`, I think you meant `because` :p

File: README.md [hunk 2]
```diff
@@ -214,12 +214,14 @@ require("telescope").extensions.rest.select_env()
...
+
 ## Lualine
 
 We also have lualine component to get what env file you select!
 And dont't worry, it will only show up under http files.
 
-```lua
+:x
```
Comment (NTBBloodbath) [inline README.md:224]: I don't think this change should be here

```suggestion
```lua
```

**Review by NTBBloodbath — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ZieMcd) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (ZieMcd) [reply README.md]: fixed

**Review by NTBBloodbath — APPROVED** [review]

**Comment by NTBBloodbath** [comment]
Hey, could you also upstream this patch to the `dev` branch README if you don't mind? Thanks :)

**Comment by ZieMcd** [comment]
sure here is pr [#296](https://github.com/rest-nvim/rest.nvim/pull/296)

**Pull request merged**